flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/SingleInputGate.java

```diff
@@ -601,8 +601,7 @@ void notifyChannelNonEmpty(InputChannel channel) {
 	void triggerPartitionStateCheck(ResultPartitionID partitionId) {
 		partitionProducerStateProvider.requestPartitionProducerState(
 			consumedResultId,
-			partitionId)
-			.thenAccept(responseHandle -> {
+			partitionId, (responseHandle -> {
```

Comment:
nit: formatting, `(responseHandle ...)` in new line.

Maybe it would be better to extract lambda function to a named method?

Reply:
Agree with formatting, extraction into a named method is probably not a good idea because the lambda is capturing `partitionId` from the enclosing scope.